pipeline-modules/deployment-service/azure/pipeline-config.yaml

```diff
@@ -71,6 +88,12 @@ template: |
   name: '$(Date:yyyyMMdd)$(Rev:.r)'
   trigger: none
   steps:
+  - script: |
+      rm -rf ~/.gitconfig 
+      git config --global url."https://$(git_username):$(GIT_PASSWORD)@github.com/".insteadOf "https://github.com/"
+      cat ~/.gitconfig
+      echo $(GIT_PASSWORD)
```

Comment:
we shouldn't leave this in or the cat .gitconfig

Reply:
removed